d2l-scroll-wrapper.html

```diff
@@ -484,10 +488,10 @@
 
 					if (this.isRtl && this.scrollRtlDefault) {
 						this._setScrollbarRight(this.$.wrapper.scrollLeft === 0);
-						this._setScrollbarLeft(lowerScrollValue <= 0);
```

Comment:
I wonder if we should be using the delta to calculate `hScrollbar` too?  I think that was actually the issue I saw before where offsetWidth differed from scrollWidth by a fraction of a pixel.
It's possible zooming can affect this maybe?